lib/tidewave/configuration.rb

```diff
@@ -0,0 +1,12 @@
+module Tidewave
+  class Configuration
+    attr_accessor :logger, :allowed_origins, :localhost_only, :allowed_ips
+
+    def initialize
+      @logger = Logger.new(STDOUT)
+      @allowed_origins = nil
+      @localhost_only = nil
```

Comment:
```suggestion
      @localhost_only = true
```

Reply:
I put it like that to read default from `fast-mcp` but maybe you are right and it would be better to have a safe default here.

Reply:
Yes, exactly regardless of their default we are sure it is true here.